salt/modules/aptpkg.py

```diff
@@ -1666,7 +1666,7 @@ def get_repo(repo, **kwargs):
     ppa_auth = kwargs.get('ppa_auth', None)
     # we have to be clever about this since the repo definition formats
     # are a bit more "loose" than in some other distributions
-    if repo.startswith('ppa:') and __grains__['os'] in ('Ubuntu', 'Mint', 'neon'):
+    if repo.startswith('ppa:') and __grains__['os'] in ('Ubuntu', 'Mint', '3000'):
```

Comment:
this should go back to neon for same reasons as above, there is an actual kde release named neon